sdk/containerservice/Azure.ResourceManager.ContainerService/api/Azure.ResourceManager.ContainerService.netstandard2.0.cs

```diff
@@ -847,6 +865,24 @@ public partial class ContainerServiceUserAssignedIdentity
         public override string ToString() { throw null; }
     }
     [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct KeyVaultNetworkAccessType : System.IEquatable<Azure.ResourceManager.ContainerService.Models.KeyVaultNetworkAccessType>
```

Comment:
Add a prefix?

Reply:
Rename to `ManagedClusterKeyVaultNetworkAccessType`